backend/app/utils/config_definitions/utils.py

```
@@ -6,6 +6,8 @@
 
 """
 
+from fastapi import Request
```

Ruff (0.8.0)

`fastapi.Request` imported but unused

Remove unused import: `fastapi.Request`

(F401)